README.md

```diff
@@ -121,6 +123,17 @@ Safari browser. To redirect back to the app, the app specifies a
 custom scheme before launching the browser. If the browser is 
 redirected to the custom scheme the browser will exit and give
 the URL back to the app.
+JJH: did you consider using a claimed https url instead of custom scheme? Both will work, but I have a feeling the claimed https url has better security properties, and I can't see any advantage to a custom scheme.
```

Comment:
I totally agree with you but I thought we have to use a custom scheme with the ASWebAuthenticationSession. I can't test it because I don't have an iOS Developer Account and I think I need one to set up a Universal Link. If using a Universal Link is possible, we should change the description here!

Reply:
I can give you access to my iOS test environment (see my message on the OSW2020 slack).

Reply:
(and it does work without custom schemes, my test apps use only claimed https urls)

Reply:
I'll update the text here.